.github/workflows/docker-build.yaml

```diff
@@ -10,6 +10,8 @@ jobs:
     timeout-minutes: 45
     steps:
       - uses: actions/checkout@v4
+      - name: Build the function
+        run: docker build -t test_function:latest --build-arg TARGETARCH="amd64" -f ./function/Dockerfile .
```

Comment:
Is it possible to move this Dockerfile inside `tests/basic` and not use `root` folder?

Reply:
I would maybe change the name too as we did with `Sample-Docker` to avoid interactions with our CI from SPS.